extensions/scarb-doc/src/db.rs

```diff
@@ -31,14 +32,19 @@ pub struct ScarbDocDatabase {
 }
 
 impl ScarbDocDatabase {
-    pub fn new(project_config: Option<ProjectConfig>) -> Self {
+    pub fn new(
+        package_name: &String,
+        project_config: Option<ProjectConfig>,
+        features_cfg: CfgSet,
+    ) -> Self {
         let mut db = Self {
             storage: Default::default(),
         };
 
         init_files_group(&mut db);
 
-        db.set_cfg_set(Self::initial_cfg_set().into());
+        let initial_cfg = Self::initial_cfg_set();
+        db.set_cfg_set(initial_cfg.into());
```

Comment:
Do you mean that i created separate variable for that or why do we set the global Cfg for db? Because I'm wondering rn if we even need this global cfg at all

Reply:
The first one. And yes, we need the global config, if a package has `cfg_set: None` then it will be used